grails-app/controllers/org/pih/warehouse/picklist/PicklistController.groovy

```diff
@@ -174,4 +178,34 @@ class PicklistController {
                 throw new IllegalFormatException("Unable to determine the proper rendering format for request for format ${format}")
         }
     }
+
+    def importPickListItems(ImportDataCommand command) {
+        Location location = Location.get(session.warehouse.id)
+
+        StockMovement stockMovement = stockMovementService.getStockMovement(params.id)
+        List<PickPageItem> pickPageItems = stockMovementService.getPickPageItems(params.id, null, null, false)
```

Comment:
I think if you pass an additional argument (stepNumber) to getStockMovement(id, stepNumber) you should get the pickPageItems populated, no?

Reply:
Yeah, that's weird. I just remembered that handle the GET stock movement items request via a separate API. So maybe we used to populate the stock movement items (pick page items, etc) from within that method, but needed to pull it out into a separate API endpoint when we added pagination support for the stock movement items API. 

:shrug: